code/modules/surgery/organs/external/_external_organs.dm

```diff
@@ -438,6 +438,9 @@
 	return GLOB.pod_hair_list
 
 /obj/item/organ/external/pod_hair/override_color(rgb_value)
+	if(ishuman(owner))
+		var/mob/living/carbon/human/H = owner
+		return H.hair_color
 	var/list/rgb_list = rgb2num(rgb_value)
```

Comment:
Is this part ever reached? Pod hair owners are only ever humans.

Reply:
This was mostly me copy-pasting from a different override_color proc without fully understanding what was going on under the surface. I assume the if statement can be safely removed.

Reply:
Just remove the bits after the if statement.

Reply:
Done.